src/integration-test/java/apptest/ReadDetailsIT.java

```diff
@@ -87,7 +69,7 @@ void test4_readDetailsByCustomerEngagementOrgId() throws Exception {
 	}
 
 	@Test
-	void test5_readDetailsByCustomerEngagementOrgIdNotFound() throws Exception {
+	void test4_readDetailsByCustomerEngagementOrgIdNotFound() throws Exception {
```

Comment:
This test and test2_readDetailsByPartyIdNotFound looks very similar, do they not? Remove one of them?